kinto_emailer/__init__.py

```diff
@@ -161,7 +170,7 @@ def includeme(config):
 
     # Listen to collection modification before commit for validation.
     config.add_subscriber(_validate_emailer_settings, ResourceChanged,
-                          for_resources=('collection',),
+                          for_resources=('bucket', 'collection',),
                           for_actions=('create', 'update'))
 
     # Listen to collection and record change events.
```

Comment:
I guess we should probably support groups and buckets events for email notifications.